TwitchPlaysAssembly/Src/UI/ModuleCameras.cs

```diff
@@ -427,16 +443,19 @@ public void SetNotes()
 					(numNameless > 0 ? $"{numNameless} item{(numNameless == 1 ? "" : "s")}" : "") +
 					(numNameless > 0 && numNamed > 0 ? " + " : "") +
 					TwitchGame.Instance.CommandQueue.Where(c => c.Name != null).Select(c => $"“{c.Name}”").Join(", ");
-				NotesTextBackgrounds[ix].color = Color.HSVToRGB(hue, .246f, .93f);
+				if (!TwitchPlaySettings.data.DarkMode) NotesTextBackgrounds[ix].color = Color.HSVToRGB(hue, .246f, .93f);
+				else NotesTextBackgrounds[ix].color = new Color32(0x3A, 0x3A, 0x3D, 0xFF);
```

Comment:
This needs to maintain the hue of TP even in dark mode. Otherwise it's hard to know if whitelist is enabled.